cookies/forms.py

```diff
@@ -279,3 +279,29 @@ class Meta:
 
 class ConceptEntityLinkForm(forms.Form):
     uri = forms.CharField(max_length=255, help_text='Manually enter a ConceptPower URI', label='URI')
+
+
+class UserAddCollectionForm(forms.ModelForm):
+
+    """
+    Form for allowing Curator to create a collection from
+    the collection list view.
+    """
+
+    uri = forms.CharField(**{
+        'required': False,
+    })
+
+    class Meta:
+        model = Collection
+        fields = ['name', 'public', 'uri', 'content_type',
+        'content_resource', 'resources']
+        widgets = {
+            'resources': forms.CheckboxSelectMultiple(),
```

Comment:
Actually, I'm inclined to have you exclude the ``resources`` field -- this is a huge queryset, so we'll need a different mechanism for adding resources to collections.